commands/publish/command.js

```diff
@@ -110,6 +110,15 @@ exports.builder = (yargs) => {
       describe: "Verify package read-write access for current npm user.",
       type: "boolean",
     },
+    "summary-file": {
+      // generate lerna publish json output.
+      type: "string",
+    },
+    // y: {
```

Comment:
Why was this commented code added?

Reply:
Got lost in the merge! I've removed and updated this in b0a9cc7